### PR TITLE
Update Ktor

### PR DESCRIPTION
**Pull request by saltanovas**
According to [OWASP Gradle dependency scan](https://github.com/dependency-check/dependency-check-gradle), Ktor 1.6.5 version has 2 vulnerabilities:

image

The next version after Ktor 1.6.5 is breaking, meaning that I updated the major library version.

### CODE REVIEW

File: src/main/kotlin/eu/kevin/api/models/payment/paymentStatus/GetPaymentStatusRequest.kt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package eu.kevin.api.models.payment.paymentStatus
 
-data class GetPaymentStatusRequest @JvmOverloads constructor(
+data class GetPaymentStatusRequest(
```
Comment (saltanovas) [inline src/main/kotlin/eu/kevin/api/models/payment/paymentStatus/GetPaymentStatusRequest.kt:3]: ‘@JvmOverloads’ annotation has no effect for methods without default arguments

File: src/main/kotlin/eu/kevin/api/Dependencies.kt [hunk 2]
```diff
@@ -22,21 +22,22 @@ import java.time.LocalDateTime
 object Dependencies {
     val httpClient by lazy {
         HttpClient(CIO) {
-            install(JsonFeature) {
-                serializer = KotlinxSerializer(Dependencies.serializer)
+            install(ContentNegotiation) {
+                json(serializer)
+                expectSuccess = true
```
Comment (saltanovas) [inline src/main/kotlin/eu/kevin/api/Dependencies.kt:27]: > With v2.0.0, the expectSuccess property used for [response validation](https://ktor.io/docs/response-validation.html) is set to false by default. If you [handle non-2xx exceptions](https://ktor.io/docs/response-validation.html#non-2xx) using handleResponseExceptionWithRequest, you also need to enable expectSuccess explicitly.

Source: https://ktor.io/docs/migrating-2.html#response-validation

File: README.md [hunk 3]
```diff
@@ -20,13 +20,13 @@ Package and installation instructions are available at the [Maven Central Reposi
...
   <artifactId>kevin-jvm</artifactId>
-  <version>0.3.0</version>
+  <version>1.0.0</version>
 </dependency>
 ```
 
 ### Gradle
 ```
-implementation 'eu.kevin:kevin-jvm:0.3.0'
+implementation 'eu.kevin:kevin-jvm:1.0.0'
```
Comment (tkopalinskas) [inline README.md:29]: Why version 1?

Comment (saltanovas) [reply README.md:30]: I believe it's breaking. If someone is already using Ktor 1.6.x in their project or Ktor 1.6.x is coming from another dependency, it's compatibility issue

**Review by stosik — APPROVED** [review]

File: src/main/kotlin/eu/kevin/api/services/account/AccountClient.kt [hunk 4]
```diff
@@ -14,28 +14,26 @@ import eu.kevin.api.models.account.list.AccountResponse
 import eu.kevin.api.models.account.transaction.request.GetAccountTransactionsRequest
 import eu.kevin.api.models.account.transaction.response.AccountTransactionResponse
 import io.ktor.client.*
+import io.ktor.client.call.*
```
[the remark this answers is not in the document]
Comment (saltanovas) [reply src/main/kotlin/eu/kevin/api/services/account/AccountClient.kt:17]: IT's actually disabled: 

```
disabledRules.set(
        setOf(
            // can't be automatically fixed and does not affect code readability
            "no-wildcard-imports"
        )
```

**Pull request merged**